install.md

```diff
@@ -16,10 +16,12 @@ If you have problems when running Podman in [rootless](README.md#rootless) mode
 sudo yum -y install podman
 ```
 
-#### [Fedora-CoreOS](https://coreos.fedoraproject.org), [Fedora SilverBlue](https://silverblue.fedoraproject.org)
+#### [Fedora-CoreOS](https://coreos.fedoraproject.org), [Fedora SilverBlue](https://silverblue.fedoraproject.org), [Endless OS](https://endlessos.com)
```

Comment:
Please add an extra entry for Endless OS. openSUSE Kubic has the has a dedicated one as well.